board/miyoo/rootfs/etc/main

```diff
@@ -56,6 +56,21 @@ restore_temp_log_func(){
  && echo "Did not find valid ${HOME}/options.cfg file, using default configuration." >> ${LOGS}
 
 # load kernel modules
+# Load sound modules
```

Comment:
a lot of stretch for alone sound module to work, could we leave it built-in instead and is that much of a gain if modulized?

Reply:
yes, is crucial as it also use dma memory